requirements/requirements.txt

```diff
@@ -1,4 +1,4 @@
-torch==1.10.0
+torch>=1.10.0
```

Comment:
Do we want to use `torch>=1.10.0,<1.11.0`?

I'm not aware of any (intended) backward-incompatible changes in PyTorch 1.11, but using a min/max convention can guard against unexpected breaks until we have a chance to test new major versions.

Reply:
yeah good point.